cylc/flow/scripts/cylc_ext_trigger.py

```diff
@@ -86,7 +86,7 @@ def main(parser, options, suite, event_msg, event_id):
         try:
             pclient(
                 'put_ext_trigger',
```

Comment:
Migrate to `graphql`? Probably a topic of a different PR

Reply:
Converting the CLI to graphql is going to have to be an alpha3 or beta1 feature.